docs/learning/tutorials/develop-web-viewer.md

```diff
@@ -1,19 +1,25 @@
 ## Developing a web viewer
 
-### Setup
-- [Install necessary prerequisites]($docs/getting-started/development-prerequisites)
-- [Clone imodeljs-samples repo](https://github.com/imodeljs/imodeljs-samples)
-- Have access to a cloud hosted iModel. If you do not have access to one, follow one of our tutorials to [create an online iModel]($docs/learning/tutorials/index.md)
+### Develop
+- [Install necessary prerequisites]($docs/getting-started/development-prerequisites).
+- From a terminal, `npx create-react-app your-app-name --template @bentley/itwin-viewer --scripts-version @bentley/react-scripts`.
+    - This will generate a new application based on the iTwin Viewer React component in the `your-app-name` directory.
+- Open the `your-app-name` directory in VS Code.
+- Add a valid contextId and iModelId for your user to the REACT_APP_TEST_CONTEXT_ID and REACT_APP_TEST_IMODEL_ID variables in the .env file within the application's root directory.

```

Comment:
Can you add a note that these IDs can be retrieved from the "Guids" column of the [registration dashboard](https://www.itwinjs.org/getting-started/registration-dashboard?tab=1)

Reply:
Added